runner/dl_checkpoints.sh

```diff
@@ -106,6 +107,17 @@ function download_sam2_checkpoints() {
 function download_stable_diffusion_checkpoints() {
     huggingface-cli download KBlueLeaf/kohaku-v2.1 --local-dir models/ComfyUI--models/checkpoints --include "*.safetensors"
     huggingface-cli download stabilityai/sd-turbo --local-dir models/ComfyUI--models/checkpoints --include "*.safetensors"
+
+    # ComfyUI_StreamDIffusion is loading single file safetensors from /models/checkpoints
+    wget -P models/checkpoints/ https://huggingface.co/KBlueLeaf/kohaku-v2.1/resolve/main/kohaku-v2.1.safetensors
+    wget -P models/checkpoints/ https://huggingface.co/stabilityai/sd-turbo/resolve/main/sd_turbo.safetensors
+    # chenxxiao/3dCartoonVision_v10
+    wget -P models/checkpoints https://huggingface.co/chenxxiao/3dCartoonVision_v10/resolve/main/3dCartoonVision_v10.safetensors?download=true --content-disposition
```

Comment:
```suggestion
    huggingface-cli download KBlueLeaf/kohaku-v2.1 --local-dir models/checkpoints --include "*.safetensors"
    huggingface-cli download stabilityai/sd-turbo --local-dir models/checkpoints --include "*.safetensors"

    # chenxxiao/3dCartoonVision_v10
    huggingface-cli download chenxxiao/3dCartoonVision_v10 --local-dir models/checkpoints --include "*.safetensors"
```

1. For the sake of consistency, can we change everything to get downloaded with `huggingface-cli`? This what I suggested worked for me
2. Do I understand correctly that the streamdiffusion node takes models from `/models/checkpoints` and not from the comfyui workspace directory `/comfui/models`?

Reply:
Created HF-hub repos for checkpoints and Loras to get rid of wget downloads.

The streamdiffusion node takes models the comfyui workspace directory `/comfui/models/checkpoints`.